lib/connectors/graphite.js

```diff
@@ -49,7 +48,7 @@ export default class Graphite {
           return reject(err);
         }
 
-        debug('[GRAPHITE] ' + JSON.stringify(metric));
+        console.log('[GRAPHITE] ' + JSON.stringify(metric));
```

Comment:
We should remove this line. It will be too verbose, we don't want a log for each graphite metric